server/src/rooms/data/roomData.json

```diff
@@ -116,10 +116,10 @@
     "hidden": true
   },
   "screen2": {
-    "displayName": "Screen 2",
-    "shortName": "Screen 2",
+    "displayName": "Screen 2: Attack of the 50 Foot Goblin",
+    "shortName": "Attack of the 50 Foot Goblin",
     "id": "screen2",
-    "description": "You enter a pitch black space you can only assume must be a theater with a screen at one end and many seats. You see nothing, however, beyond a blackness that cannot possibly be natural. You hear wet noises coming from the back corner of this room, and you fear to wonder what might be their source.<br/><br/>Or you can head back to the [[main theater->theater]].",
+    "description": "As you walk into the theater, a goblin wearing a bikini walks into a visibly fake power line, crushing it in her hands. The camera cuts to a goblin cafe as a couple makes out under flickering lights, and complains about the disruption.<br/><br/>Or you can head back to the [[main theater->theater]].",
```

Comment:
As you walk into the theater, on the screen a goblin wearing a bikini walks into a visibly fake power line, crushing it in her hands. The camera cuts to a goblin cafe as a couple makes out under flickering lights, and complains about the disruption.<br/><br/>You can take a seat here, or you can head back to the [[main theater->theater]].

Reply:
(clarify the goblin is on screen, and fix the room link being a dangling sentence)